### PR TITLE
Nerfs peaceborg chain bucklestun

### PR DESCRIPTION
**Pull request by AnturK**
Why was this ever a thing ?

### CODE REVIEW

**Comment by Arianya** [comment]
Oranges why are you on Antur's account

File: code/game/objects/items/robot/robot_items.dm [hunk 1]
```diff
@@ -561,13 +561,29 @@
...
+	if(owner)
+		owner.module.allow_riding = TRUE
+
+/obj/item/borg/projectile_dampen/proc/get_host()
+	if(istype(host))
+		return host
+	else
+		if(iscyborg(host.loc))
+			return host.loc
+	return
```
Comment (RemieRichards) [inline code/game/objects/items/robot/robot_items.dm]: ...

Comment (AnturK) [reply code/game/objects/items/robot/robot_items.dm]: Oh duh was going to be explicit null.

Comment (AnturK) [reply code/game/objects/items/robot/robot_items.dm]: To be honest i should have put that proc somewhere in the root of /borg items

**Comment by Anonmare** [comment]
Feature

**Comment by jughu** [comment]
Wrong solution.

Good solution = removing borgs instabuckling you and auctally add a mount time like piggybacking

**Comment by kevinz000** [comment]
👍 right solution

**Comment by DoctoreMad** [comment]
Peaceborg is basically a meme borg now. The two things it has, dispensing cookies and slowing down people, is superceded by Medborgs that can dispense healing sweets and administer stun chems.
Just throw them away already.

**Comment by Anonmare** [comment]
Why not just make them capable at their job?

**Comment by Supermichael777** [comment]
fug sec XXXXXXXD ^

**Comment by Arianya** [comment]
The running "joke" is that peacborgs are a trap option to "catch out" secborg players because obviously secborgs were literally satan and this justifies having a trap option in borg module selection.

And worse, its a maintainer backed "joke"

**Comment by RemieRichards** [comment]
>trap option

just git gud?

**Comment by Arianya** [comment]
The definition of a trap option is that you don't know its a trap option until you use it. Obviously anyone who has tried it before realizes that they're a stunted joke but that doesn't make it any worse of a addition to the game.

File: code/game/objects/items/robot/robot_items.dm (hunk 1 above)
Comment (optimumtact) [reply code/game/objects/items/robot/robot_items.dm]: d

**Comment by optimumtact** [comment]
you sound upset

File: code/game/objects/items/robot/robot_items.dm (hunk 1 above)
Comment (AnturK) [reply code/game/objects/items/robot/robot_items.dm]: Oh, forgot about this one, fixing in a sec.

**Comment by Incomptinence** [comment]
The hilarious thing is initial peace keeping borg with bola launcher was ""good"" enough to trigger anti borg salt again and get the bola launcher removed.

**Comment by Arianya** [comment]
@optimumtact There's a difference between salt and bad game design

**Pull request merged**